Makefile

```diff
@@ -9,6 +9,8 @@ GOFILES = $(shell find . -type f -name '*.go')
 packetbeat: $(GOFILES)
 	# first make sure we have godep
 	go get github.com/tools/godep
+	# gotestcover is needed to fetch coverage for multiple packages
+	go get github.com/pierrre/gotestcover
```

Comment:
I'd rather put this one in the `cover` target, because it's not needed for building the binary.